### ISSUE TITLE
xcodebuild: error: 'surge-vst2.xcodeproj' does not exist.

### ISSUE THREAD

**Issue by esaruoho** [kurasu/surge#46]
getting this error when running `build-osx.sh`

```
xcodebuild: error: 'surge-vst2.xcodeproj' does not exist.
```

not sure how to proceed?

**Comment by kzantow** [comment]
Actually, what you need to do _before_ running `premake5 xcode4` is to set the environment variable `VST2SDK_DIR`, e.g.: `export VST2SDK_DIR=~/programming/VST_SDK_2.4`, then run `premake5 xcode4` and you'll get a VST2 project.

**Comment by kzantow** [comment]
@esaruoho yes. it's been end-of-life'd from steinberg, I don't think you can get it from them anymore and they said something like it's against the EULA to contiue to use it to build plugins if you haven't signed a developer agreement with them in the past.

**Comment by kzantow** [comment]
I signed the developer agreement years ago, if I recall. I know at least I tried to on multiple occasions, barring website issues.

**Comment by esaruoho** [comment]
@kzantow what are your thoughts re: whether it makes sense to focus on getting VST2 to compile on Linux,macOS,Windows, or should just go VST3 from the start? Or might VST2 be easier to get to compile?

**Comment by kzantow** [comment]
@esaruoho the problem with VST3 is that many hosts still don't support it, so VST2 is still the lowest common denominator. I've got it compiling, just that odd issue on OSX indicating the architecture is not right: #3
